apps/3000-home/remote-delegate.js

```diff
@@ -1,19 +1,17 @@
-// import { importDelegatedModule } from '@module-federation/utilities';
+import { importDelegatedModule } from '@module-federation/utilities/src/utils/importDelegatedModule';
```

Comment:
Why is this file restored to abnormal use

Reply:
This file can be removed now, ill clean it up. 
before it was to avoid eager modules from export * getting require()'d so i have to import the "pure" module directly.